### PR TITLE
[skip actions] NH-27837-c: create verify_install workflow (template)

### PR DESCRIPTION
**Pull request by xuan-cao-swi**
## Why?
For verify_install action on main (dispatch run).

### CODE REVIEW

File: .github/workflows/verify_install.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  workflow_dispatch:
+
+
+env:
+  SW_APM_SERVICE_KEY: ${{ secrets.SW_APM_SERVICE_KEY_PROD }}
+  SW_APM_COLLECTOR: ${{ secrets.SW_APM_COLLECTOR_PROD}}
+
+jobs:
+  #--------------------------------------------------------------------
+  # RUBY 3.0
```
Comment (cheempz) [inline .github/workflows/verify_install.yml:13]: Nit: should be 3.1?

File: .github/workflows/verify_install.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  ruby31_install_ubuntu_20_arm:
+    runs-on: ubuntu-latest
+    container:
+      image: arm64v8/ruby:3.1.3-bullseye
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```
Comment (cheempz) [inline .github/workflows/verify_install.yml:38]: Why is docker buildx action needed?

**Review by cheempz — COMMENTED** [review]
Thanks @xuan-cao-swi, I left a few comments.

File: .github/workflows/verify_install.yml (hunk 2 above)
Comment (xuan-cao-swi) [reply .github/workflows/verify_install.yml:38]: removed.

File: .github/workflows/verify_install.yml (hunk 1 above)
Comment (xuan-cao-swi) [reply .github/workflows/verify_install.yml:13]: I was thinking to have ruby version 3 on same section, but I changed it to 3.1

**Pull request merged**